deploy/examples/object.yaml

```
@@ -105,16 +105,16 @@ spec:
     #       secret:
     #         secretName: rgw-ldap
     #         defaultMode: 0600
-  #zone:
-  #name: zone-a
-  # service endpoint healthcheck
     # opsLogSidecar:
     #   resources:
     #     limits:
     #     memory: "60Mi"
     #   requests:
     #     cpu: "100m"
     #     memory: "60Mi"
+  #zone:
+  #name: zone-a
+  # service endpoint healthcheck
```

Let's take some time to fix the formatting of the comment here as well to match the rest of the example, and to fix the broken indentation of `name`.

```suggestion
  # zone:
  #   name: zone-a
  # service endpoint healthcheck
```